src/free.c

```diff
@@ -291,27 +291,27 @@ dwg_free_xdata(Dwg_Object_XRECORD *obj, int size)
   obj->xdata = NULL;
 }
 
-static void
+void
```

Comment:
you need to EXPORT it then

Reply:
this broke decode_test:
/usr/bin/ld: decode_test.o: in function `dwg_decode_eed':
/home/rurban/Software/libredwg/test/testcases/./../../src/decode.c:2428: undefined reference to `dwg_free_eed'
/usr/bin/ld: /home/rurban/Software/libredwg/test/testcases/./../../src/decode.c:2413: undefined reference to `dwg_free_eed'
clang: error: linker command failed with exit code 1 (use -v to see invocation)